### ISSUE TITLE
Revisions to standalone header feature

### ISSUE THREAD

**Issue by dananjohnson** [ManifoldScholar/manifold#2304]
- [ ] apply shadow on standalone header to library header as well
- [ ] standardize background color on dropdown menus

**Comment by zdavis** [comment]
Comps:
![image](https://user-images.githubusercontent.com/255909/62725578-42fe3880-b9ca-11e9-9ac4-4d4ecad0e5d5.png)

Implementation:
![image](https://user-images.githubusercontent.com/255909/62725604-51e4eb00-b9ca-11e9-9fd2-ebb4c54d0df4.png)

**Comment by tsmyre** [comment]
The Standalone mode tool tip reads incorrectly at present:

> If enabled, this project will always render in standalone mode when “&mode=standalone” is appended to the URL.

The ampersand should be a question mark: `?mode=standalone`.

**Comment by tsmyre** [comment]
Also, when toggled to Enabled a project does not persist the `?mode=standalone` in the URL when a user clicks on a text. For example, see this project: `https://edge.manifoldapp.org/projects/the-adventures-of-sherlock-holmes?mode=standalone`.  

If you click on any of the texts and enter the reader, the menu you see will still provide all the options available for library view: Project Home, Projects, Following. If you nav back that way, Standalone is no longer visible.

Likewise, if you return fresh to `https://edge.manifoldapp.org/projects/the-adventures-of-sherlock-holmes?mode=standalone` and click on any resource, when you arrive at the resource page the standalone suffix will be gone, though the standalone header will persist. If you use that to nav back to the project home, the standalone header will be there even though the URL suffix is not showing. However if you click refresh, you'll be out of standalone.

**Comment by zdavis** [comment]
![image](https://user-images.githubusercontent.com/255909/63063602-2c684d80-beb2-11e9-9677-835cd4f1b1ab.png)

**Comment by tsmyre** [comment]
I'm wondering if there should be a default header bar color when a user doesn't specify one in settings. Right now the bar turns white and is only visible on hover if no color is provided.

**Comment by tsmyre** [comment]
I'm also wondering if it would be possible to have a toggle that allows a public project to be in standalone and not appear in the library. I suspect that will be a highly sought after means of conveying projects when in standalone.

**Comment by dananjohnson** [comment]
> I'm wondering if there should be a default header bar color when a user doesn't specify one in settings. Right now the bar turns white and is only visible on hover if no color is provided.

Yeah there definitely needs to be a default color here. Without one specified, the BG is actually just transparent.

<img width="1444" alt="Screen Shot 2019-08-22 at 9 08 52 AM" src="https://user-images.githubusercontent.com/12454203/63531085-9e492400-c4bc-11e9-817c-aefa23ff4d57.png">

**Comment by zdavis** [comment]
Could you implement this, Dana? I say we go with the manifold green as default.

**Comment by dananjohnson** [comment]
Can do! Although that green is also the hover color. I'm wondering if we should make green the default, but update the hover/focus state to just lighten or darken whatever the BG color is.

**Comment by zdavis** [comment]
I like that idea

**Comment by zdavis** [comment]
![image](https://user-images.githubusercontent.com/255909/64725970-8563db00-d48a-11e9-8f3e-3aafd408558d.png)
